docs/user-management.md

```diff
@@ -0,0 +1,87 @@
+
+## New User Interface
+
+The BC Developer Registry has undergone significant UI improvements to provide a more intuitive and user-friendly experience. You can now easily navigate through the website, access different sections, and manage your projects and users more efficiently.
+
+## User Management Feature
+
+One of the key updates is the introduction of a comprehensive user management feature. This feature allows product owners and technical leads to manage AWS permissions for users associated with their product. Users can be assigned to one or more of the following roles:
+
+1. **Viewers**
+2. **Billing Viewers**
+3. **Developers**
+4. **Admins**
+5. **Security Auditors**
+
+![Registry Roles](images/user-management/registry_roles.png)
+### Prerequisites
+
+- Users must have a valid IDIR to be added to these roles.
+- Only product owners or technical leads can view and manage users in the website.
+
+### Role-Based Access
+
+Assigning users to these roles grants them specific permissions to AWS accounts associated with those roles. Here's a breakdown of the roles and their corresponding AWS policies:
+
+#### Admins
+
+- **Policy**: `arn:aws:iam::aws:policy/AdministratorAccess`
+- Admins have full access to all four AWS accounts within the product: dev, test, prod, and tools.
+
+#### Developers
+
+- **Policy**: `arn:aws:iam::aws:policy/AdministratorAccess`
+- Developers have admin access to dev, test, and tools accounts. To gain admin access to the prod account, they must have the Admin role.
+
+#### Viewers
+
+- **Policy**: `arn:aws:iam::aws:policy/ReadOnlyAccess`
+- Viewers have read-only access to the AWS accounts.
+
+#### Security Auditors
+
+- **Policy**: `arn:aws:iam::aws:policy/SecurityAudit`
+- Security Auditors have access to security configuration and logs to perform audit tasks.
+
+#### Billing Viewers
+
+- **Custom Policy**: Provides access to AWS Budgets and Cost Explorer.
+  
+```json
+{
+    "Statement": [
+        {
+            "Action": "ce:*",
+            "Effect": "Allow",
+            "Resource": "*",
+            "Sid": "AllowCostExplorer"
+        },
+        {
+            "Action": "budgets:*",
+            "Effect": "Allow",
+            "Resource": "*",
+            "Sid": "AllowBudgets"
+        }
+    ],
+    "Version": "2012-10-17"
+}
+```
+
+## Accessing the Public Cloud Landing Zone
+- Once a user is added to a role, they can access the product (AWS accounts) from the [Public Cloud Landing Zone.](https://login.nimbus.cloud.gov.bc.ca/) If users have multiple roles, they can choose any of those roles when logging in to the account.
+![Landing Zone](images/user-management/landing_zone.png)
```

Comment:
Please add an explanation of this image for accessibility purposes, describe either in list form what this images describes and provide an alt.text

https://www2.gov.bc.ca/gov/content/home/accessible-government/toolkit/accessible-digital-content/graphics